test/metabase/driver/postgres_test.clj

```diff
@@ -363,25 +407,65 @@
                                  :metabase.query-processor.util.add-alias-info/position 1}]
                 field-ordinary [:field (u/the-id field) nil]
                 compile-res (qp/compile
-                              {:database (u/the-id database)
-                               :type     :query
-                               :query    {:source-table (u/the-id table)
-                                          :aggregation  [[:count]]
-                                          :breakout     [field-bucketed]
-                                          :order-by     [[:asc field-bucketed]]}})
+                             {:database (u/the-id database)
+                              :type     :query
+                              :query    {:source-table (u/the-id table)
+                                         :aggregation  [[:count]]
+                                         :breakout     [field-bucketed]
+                                         :order-by     [[:asc field-bucketed]]}})
                 only-order  (qp/compile
-                              {:database (u/the-id database)
-                               :type     :query
-                               :query    {:source-table (u/the-id table)
-                                          :order-by     [[:asc field-ordinary]]}})]
-            (is (= (str "SELECT date_trunc('month', CAST((\"json_alias_test\".\"bob\"#>> ?::text[])::VARCHAR  AS timestamp)) "
```

Comment:
It made my eyes bleed to look at a lot of the SQL in this namespace, so I used `mdb.query/format-sql` and split lines to actually make it somewhat readable